Eniko/index.js

```diff
@@ -0,0 +1,54 @@
+// Declare two variables: "dogIsBetter" with the value of true, and "catIsBetter" with the value of false.
+let dogIsBetter = true;
+let catIsBetter = false;
+
+// Check the result of:
+
+// a) dogIsBetter AND catIsBetter.
+console.log("dog AND cat - " + (dogIsBetter && catIsBetter));
+//false - && always looks for the first (false) variable
+
+// b) dogIsBetter OR catIsBetter.
+console.log("dog OR cat - " + (dogIsBetter || catIsBetter));
+// true - || is looking for the first varible that is true
+
+// c) NOT (dogIsBetter AND catIsBetter).
+console.log("not dog and cat - " + (!(dogIsBetter && catIsBetter)));
+//true - ! changes every boolenian varible to its opposite
+
+// Declare three more variables "atoms", "sandGrains", "starsInSky". Give these variables number values.
+let atoms = 74954793;
+let sandGrains = 847297034;
+let starsInSky = 36674393;
+
+// Check the result of whether:
+
+// a) atoms is greater than starsInSky AND atoms is greater than sandGrains.
+console.log("atoms is greater than starsInSky AND atoms is greater than sandGrains " + (atoms > starsInSky));
```

Comment:
Also check if atom is also greater than starsInSky!!!
console.log(`Q3a: ${(atoms > starsInSky) && (atoms > sandGrains)}`);